config/config.go

```diff
@@ -6,6 +6,8 @@ import "time"
 type Config struct {
 	// APIKey is available from https://account.gandi.net/en/
```

Comment:
Explain this is deprecated by Gandi.
```suggestion
	// APIKey has been deprecated by Gandi in favor of Personal Access Token. Please refer to the Gandi authentication documentation: https://api.gandi.net/docs/authentication/ for details.
```